requirements.txt

```diff
@@ -5,7 +5,7 @@ ConfigArgParse==1.3
 distro==1.7.0
 setuptools==65.5.1  # For pkg_resources
 six==1.16.0
-packaging==21.2
+packaging==23.1
```

Comment:
Needed for new `black`, anyway in gProfiler code we use only `packaging.version.Version` which I suppose continues to function properly.